blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/Configuration.java

```diff
@@ -65,6 +65,20 @@ public void init() throws IOException {
         return Collections.unmodifiableMap(props);
     }
 
+    public Map<Object, Object> getAllProperties() {
```

Comment:
I must be missing something. What does it do differently than getProperties? As far as I can see what getProperties returns should have default values in them. (init() method gets called during instantiation which seem to populate with default values first and then override them)